### ISSUE TITLE
Clients should be able to set their own homescreen/navbar icons

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#456]
From #445 

What do we want to support, links, base64 encoding? Whatever it is will need to be able to passed across the iframe. What do you think @samreid?

**Comment by samreid** [comment]
I tried searching Google for examples of using postMessage to send images and couldn't find anything.  base64 sounds like it would work but before resorting to that we should try sending higher-level representations such as canvas or Image to see if it works as desired.

**Comment by samreid** [comment]
We added support for setting images using instance proxies, but @zepumph and I thought we should focus on Faraday's Law (a single screen sim without any icons) until it is ready first.  Putting this issue on hold.

**Comment by samreid** [comment]
@kathy-phet and @ariel-phet can you help prioritize/schedule work for this issue?  It could take around 4 hours for a rough draft/initial investigation.

**Comment by ariel-phet** [comment]
@samreid this strikes me very much as a "wishlist" type feature. I think best to revisit when Faraday's Law is complete.

**Comment by pixelzoom** [comment]
@samreid @ariel-phet Why are we considering this? Has someone asked for this?  Even if they asked, does PhET want clients changing screen icons?  What other graphics will they want to change next?

**Comment by ariel-phet** [comment]
@samreid @zepumph in https://github.com/phetsims/joist/issues/445#issue-264417237 there is the following item

> Should client be able to remove the navbar? Kathy says not until someone asks for it.
